.stylelintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "stylelint-config-recommended-scss",
+  "rules": {
+    "no-descending-specificity": null,
+    "no-duplicate-selectors": null
```

Comment:
why do we disable these?
when `no-duplicate-selectors` is giving an error, it's most likely a developer error.
`no-descending-specificity` is good for readability, but somewhat annoying , so I don't have a strong opinion about it